Framework/Core/include/Framework/HistogramRegistry.h

```diff
@@ -228,6 +228,13 @@ class HistogramRegistry
     throw std::runtime_error("No match found!");
   }
 
+  template <typename T>
+  void fill(char const* const name, T value)
+  {
+    auto& hist = get(name);
```

Comment:
mmm... I would avoid something like this. Clearly we do not want to encourage people to fill histograms by name..

Reply:
ok, then what's the preferred option?
Sorry, I don't know what's the convention here ;-)